### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by MRXLT**
fix batch size in server 
fix output tensor

### CODE REVIEW

**Comment by MRXLT** [comment]
for issue
https://github.com/PaddlePaddle/Serving/issues/182

File: core/general-server/op/general_infer_helper.h [hunk 1]
```diff
@@ -37,14 +37,20 @@ struct GeneralBlob {
...
       tensor_vector[ti].shape.clear();
     }
     tensor_vector.clear();
   }
-  
+
+  int SetBatchSize(int batch_size) { _batch_size = batch_size; }
+
+  int GetBatchSize() const { return _batch_size; }
+  /*
```
Comment (guru4elephant) [inline core/general-server/op/general_infer_helper.h]: please remove this comment if this code is not used anymore.

File: core/general-server/op/general_infer_op.cpp [hunk 2]
```diff
@@ -36,33 +36,31 @@ using baidu::paddle_serving::predictor::InferManager;
...
   TensorVector *out = &output_blob->tensor_vector;
   int batch_size = input_blob->GetBatchSize();
+  output_blob->SetBatchSize(batch_size);
 
   VLOG(2) << "infer batch size: " << batch_size;
   // infer
-  Timer timeline;
-  double infer_time = 0.0;
-  timeline.Start();
+  // Timer timeline;
```
Comment (guru4elephant) [inline core/general-server/op/general_infer_op.cpp]: should merge https://github.com/PaddlePaddle/Serving/pull/185 after it is merged.

Comment (MRXLT) [reply core/general-server/op/general_infer_op.cpp]: conflict fixed

**Review by guru4elephant — APPROVED** [review]
LGTM

**Pull request merged**